sys/fs/fuse/fuse_vnops.c

```diff
@@ -866,19 +867,19 @@ fuse_vnop_copy_file_range(struct vop_copy_file_range_args *ap)
 	pid_t pid;
 	int err;
 
-	err = ENOSYS;
 	if (mp == NULL || mp != vnode_mount(outvp))
-		goto fallback;
+		return (EXTERROR(ENOSYS, "Mount points do not match"));
```

Comment:
This should be unreachable.  I think we should probably panic here, but that's beyond the scope of your PR.